spec/controllers/reconcilliations_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require "controller_spec_helper"
+
+RSpec.describe ReconcilliationsController do
+  before(:all) { create_users }
+
+  let(:user) { FactoryBot.create(:user) }
+  let(:facility) { FactoryBot.create(:setup_facility) }
+  let(:item) { FactoryBot.create(:item, facility: facility) }
+  let(:account) do
+    FactoryBot.create(:nufs_account, account_users_attributes: [FactoryBot.attributes_for(:account_user, user: user)]).tap do |account|
```

Comment:
We shouldn't need this full setup anymore. If you use `:setup_account`, that should do everything you need.

Reply:
Awesome! Changed in 21dff8c57